### PR TITLE
contribute notes for Raspberry Pi and Ubuntu 16.04 deployment 

### PR DESCRIPTION
**Pull request by shsmith**

### CODE REVIEW

File: contrib/python3.6/python-3.6.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Installation of Python 3.6
```
Comment (bauerj) [inline contrib/python3.6/python-3.6.sh]: This should probably be a comment.

File: contrib/raspberrypi3/run_electrumx.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+###############
+# run_electrumx
```
Comment (bauerj) [inline contrib/raspberrypi3/run_electrumx.sh]: Why is this necessary? Isn't systemd the default for Raspbian Jessie?

Comment (shsmith) [inline contrib/raspberrypi3/run_electrumx.sh]: It is not necessary. It is a tip to allow for easier testing and deployment. I'm ok if you would rather delete it.

File: contrib/python3.6/python-3.6.sh (hunk 1 above)
Comment (shsmith) [inline contrib/python3.6/python-3.6.sh]: Ok with me.

File: contrib/raspberrypi3/run_electrumx.sh (hunk 2 above)
Comment (bauerj) [inline contrib/raspberrypi3/run_electrumx.sh]: No, it's fine for me if you think it's useful. I was just confused because it looked like an init-replacement.
